lib/status/timeline.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2020 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package status
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gravitational/gravity/lib/constants"
+	"github.com/gravitational/gravity/lib/defaults"
+	"github.com/gravitational/gravity/lib/state"
+	"github.com/gravitational/gravity/lib/utils"
+
+	pb "github.com/gravitational/satellite/agent/proto/agentpb"
+	"github.com/gravitational/satellite/lib/client"
+	"github.com/gravitational/trace"
+)
+
+// Timeline queries the currently stored cluster timeline.
+func Timeline(ctx context.Context) (*pb.TimelineResponse, error) {
+	addr := fmt.Sprintf("%v:%v", constants.Localhost, defaults.SatelliteRPCAgentPort)
+
+	stateDir, err := state.GetStateDir()
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	caFile := state.Secret(stateDir, defaults.RootCertFilename)
```

Comment:
There is `lib/httplib.GetPlanetClient()` that does all this IIRC. Let's reuse that.

Reply:
Looks like `GetPlanetClient` returns an http client. We want to use the satellite rpc client here.

Reply:
Then let's create a `GetGRPCPlanetClient()` there so it can be reused and use it here.